content_scripts/vimium_frontend.coffee

```diff
@@ -580,6 +581,8 @@ window.refreshCompletionKeys = (response) ->
 
     if (response.validFirstKeys)
       validFirstKeys = response.validFirstKeys
+    if (response.commandKeys)
+      commandKeys = response.commandKeys
```

Comment:
Are you sure you've cherry picked correctly, @mrmr1993?  It looks like `commandKeys` is unused here.